backend/src/router/base.router.ts

```diff
@@ -0,0 +1,85 @@
+import express, { Application } from 'express';
+
+// https://hackernoon.com/object-oriented-routing-in-nodejs-and-express-71cb1baed9f0
+// Implemented the idea from the above post
+class Router {
+    /**
+     * Express application
+     * Needs to any so we can jank in the register Services
+     */
+    private app: any;
```

Comment:
Then you can type this back to an application
```suggestion
    private app: Application;
```